CHANGELOG.md

```diff
@@ -42,7 +42,15 @@ a subset has been implemented--see examples below) and
 - Introduced planner event callbacks as a means to provide a facility that allows the query to be visualized at every 
 stage in the `PlannerPipeline` and to generate performance metrics for the individual phases of query planning.  See
 `PlannerPipe.Builder.plannerEventCallback` for details.
-
+- Adds the following optimization passes, none of which are enabled by default:
```

Comment:
nit: for additions to the changelog related to the planner, would be good to prefix the entry w/ "[experimental]" and/or "[planner]" to make it more clear that the feature is for the experimental planner.

Alternatively, could have a separate heading for experimental planner features

Reply:
I'll move the query planner specific stuff into an h4.